src/pe_reports/charts.py

```diff
@@ -0,0 +1,162 @@
+"""Class methods for report charts."""
+
+# Third-Party Libraries
+import matplotlib.pyplot as plt
+from matplotlib.ticker import MaxNLocator
+
+
+class Charts:
+    """Build charts."""
+
+    def __init__(self, df, width, height, name, title, x_label, y_label):
+        """Initialize chart class."""
+        self.df = df
+        self.title = title
+        self.x_label = x_label
+        self.y_label = y_label
+        self.width = width
+        self.height = height
+        self.name = name
+
+    def pie(self):
+        """Build pie chart."""
+        df = self.df
+        width = self.width
+        height = self.height
+        name = self.name
+        plt.rcParams.update({"figure.max_open_warning": 0})
+        catName = df.columns[0]
+        Val_1Name = df.columns[1]
+        df = df.sort_values(by=Val_1Name, ascending=False)
+        Category_column = df[catName]
+        Value_column = df[df.columns[1]]
+        labels = Category_column
+        plt.gca().axis("equal")
+
+        # Only show the label when the value is greater than 10%.
+        def autopct(pct):
+            """Get autopct."""
+            return ("%1.0f%%" % pct) if pct > 1 else ""
+
+        pie = plt.pie(
+            Value_column,
+            startangle=0,
+            radius=1,
+            autopct=autopct,
+            textprops={"color": "w", "fontsize": 7},
+        )
+        plt.legend(
+            pie[0],
+            labels,
+            bbox_to_anchor=(1, 0.5),
+            loc="center right",
+            fontsize=7,
+            bbox_transform=plt.gcf().transFigure,
+            frameon=False,
+        )
+        plt.subplots_adjust(left=0.2, wspace=0.2)
+        plt.gcf().set_size_inches(width / 2.54, height / 2.54)
```

Comment:
The `2.54` magic number is repeated throughout this class.  You should consider replacing it with a constant.

Reply:
@dav3r Addressed in this [commit](https://github.com/cisagov/pe-reports/pull/106/commits/16db49cbf1c57112f2024e3264651e4d9c26ba32).

Reply:
Excellent, thanks 👍